js/loader.js

```diff
@@ -35,7 +35,7 @@ var Loader = (function() {
 				// This is done after all the moduels so we can
 				// overwrite all the defined styls.
 
-				loadFile("css/custom.css", function() {
+				loadFile(config.customCss, function() {
```

Comment:
I agree. It should give some feedback that it can't find the file on the provided path.

Other than that: Awesome contribution!